src/common/CMakeLists.txt

```diff
@@ -2,8 +2,6 @@ cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 
 set (sources
   math.cc
-  DepthCamera.cc
-)
+  DepthCamera.cc)
```

Comment:
Just following the same cmake code style that I see being used on other projects. I hope to add code style check on the project soon and probably add checks for the cmake files as well. Actually for long lists I prefer on a new line, but for short lists its indeed uglier compared with same line.